zio-http/shared/src/main/scala/zio/http/endpoint/Endpoint.scala

```diff
@@ -662,7 +662,7 @@ final case class Endpoint[PathInput, Input, Err, Output, Auth <: AuthType](
     alt: Alternator[Err2, Err],
   ): Endpoint[PathInput, Input, alt.Out, Output, Auth] =
     copy[PathInput, Input, alt.Out, Output, Auth](
-      error = ((ContentCodec.content[Err2]("error-response") ++ StatusCodec.status(status)) ?? doc) | self.error,
+      error = (ContentCodec.content[Err2]("error-response") ++ StatusCodec.status(status) ?? doc) | self.error,
```

Comment:
you did not revert here

Reply:
I did not revert, because that is exactly what causes the incorrect doc to be rendered - the `description` is placed under `schema` as well as under `responses.<CODE>`, which does not comply with the OpenAPI 3.0.3 spec.

Reply:
Then this is a bug in the generator. Not in the endpoint. The call is clearly: Set the content with that status and document the **whole**. The OpenAPI gen is then incorrectly adding the doc on the wrong place.
So even if this fixes the generated code, it is not the right fix.

Reply:
Right, OK, I'll take another look at it over the weekend.
Thanks for the review :)

Reply:
Found where the issue was.
Please, give this another review pass 😃